src/TextField/TextField.test.tsx

```diff
@@ -81,4 +81,36 @@ describe("TextField", () => {
 
     expect(input).toHaveValue("2256666");
   });
+
+  it("test for minRows when multiline is enabled", () => {
+    const { getByTestId } = render(<TextField multiline minRows={"2"} />);
+    const textField = getByTestId("text-field");
+
+    expect(textField).toBeInTheDocument();
+
+    // Wait for the component to finish rendering and then check the rows attribute
+    setTimeout(() => {
```

Comment:
This part is actually doing nothing. Because if I just pass as "mincRows" it passes the test. So you need some updates in this part.

Reply:
I think we cannot have tests to test minRows and maxRows there are some limitations. I just added tests to test multiline